### PR TITLE
fixing 'log into'

### PR DESCRIPTION
**Pull request by kalexand-rh**
@bobfuru, PTAL

### CODE REVIEW

File: logging/config/cluster-logging-configuring.adoc [hunk 1]
```diff
@@ -59,7 +59,7 @@ spec:
 ----
 You can configure the following for cluster logging:
 
-* You can place cluster logging into an unmanaged state that allows an administrator to assume full control of individual
+* You can place cluster logging in to an unmanaged state that allows an administrator to assume full control of individual
```
Comment (bobfuru) [inline logging/config/cluster-logging-configuring.adoc]: I don't think "into" needs to be separated in this case. Depends on whether it is referring to "cluster logging" or "cluster log in" as a concept, if that makes sense.

File: modules/ignition-config-viewing.adoc [hunk 2]
```diff
@@ -97,7 +97,7 @@ into data URLs, which tends to make the content a bit clumsy to read. (Use the
  on NFS that configures that service, you would just add an NFS configuration
  file, which would then be started by the init process when the system comes up.
 * users: A user named core is created, with your ssh key assigned to that user.
-This will allow you to log into the cluster with that user name and your
+This allow you to log in to the cluster with that user name and your
```
Comment (bobfuru) [inline modules/ignition-config-viewing.adoc]: s/allow/allows

**Review by bobfuru — COMMENTED** [review]
Couple of comments and LGTM!

File: logging/config/cluster-logging-configuring.adoc (hunk 1 above)
Comment (kalexand-rh) [reply logging/config/cluster-logging-configuring.adoc]: You're right - it doesn't need to be separated.

**Pull request merged**